supervisor/backups/backup.py

```diff
@@ -263,7 +263,7 @@ def __eq__(self, other: Any) -> bool:
 
         # Compare all fields except ones about protection. Current encryption status does not affect equality
         keys = self._data.keys() | other._data.keys()
-        for k in keys - {ATTR_PROTECTED, ATTR_CRYPTO}:
+        for k in keys - {ATTR_PROTECTED, ATTR_CRYPTO, ATTR_DOCKER}:
```

Comment:
Do we have tests to ensure these attributes are ignored?

Reply:
There's a test to confirm an encrypted backup and unencrypted backup can be consolidated, which means those fields were ignored. I didn't have `docker` in there before so my backup fixtures didn't have that, I'll have to check on that.

Technically for docker we should only be ignoring it if one backup is encrypted and one isn't, otherwise it should be the same. I don't know if we want to deal with that for a field to be deprecated though

Reply:
> I don't know if we want to deal with that for a field to be deprecated though

Yeah let's not bother about that.

But adding `docker` attributes and make sure they get ignored in the test make sense :+1: .

Reply:
Ok, I've updated the test fixture files to include a docker repository setting. The test fails without the change above, but with the change merged succeeds :+1: .